src/index.html

```diff
@@ -129,40 +129,47 @@ <h5 class="timeline-show-less">Hide Previous Clubs</h5>
 
       <section id="sponsors-partners" class="bg-grey col-12 py-5">
         <h2 class="display3 text-center mb-5">A Huge Thank You to Our Sponsors and Partners</h3>
-          <h3 class="display4 text-center mb-4">Sponsors</h4>
-            <div class="col-10 d-flex flex-wrap justify-content-center mx-auto mb-5 align-items-center">
-              <a href="http://mentorsfound.com" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/sponsors/mentors.png" alt="Mentors Found">
-              </a>
-              <a href="http://www.hudl.com" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/sponsors/hudl.png" alt="Hudl">
-              </a>
-              <a href="http://www.assurity.com/" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/sponsors/assurity.png" alt="Assurity">
-              </a>
-            </div>
-            <h3 class="display4 text-center mb-4">Partners</h3>
-            <div class="col-10 d-flex flex-wrap justify-content-center mx-auto align-items-center">
-              <a href="http://www.launchleadership.org" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/partners/launch.png" alt="Launch Leadership">
-              </a>
-              <a href="http://www.fusecoworking.com/" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/partners/fuse.png" alt="FUSE Coworking">
-              </a>
-              <a href="http://makeshiftlincoln.org/" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/partners/makeshift.png" alt="Makeshift Lincoln">
-              </a>
-              <a href="https://www.selfimagedefined.org/" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/partners/selfid.png" alt="Self Image Defined">
-              </a>
-              <a href="http://leturaidigima.com" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/partners/letura.png" alt="Letura Idigima">
-              </a>
-              <a href="http://www.carlosvelascodesign.com" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/partners/carlos.png" alt="Carlos Velasco Design">
-              </a>
-            </div>
-            </sction>
+          <h3 class="display4 text-center mb-4">Sponsors</h3>
```

Comment:
I had some mismatched tags in here that I just caught.

previously had <h3></h4> and <section></sction>